closed/autoconf/custom-spec.gmk.in

```diff
@@ -121,3 +121,9 @@ ifeq (true,$(OPENJ9_ENABLE_CMAKE))
 else
   OPENJ9_VM_BUILD_DIR := $(OUTPUTDIR)/vm
 endif
+
+ifeq (riscv64,$(OPENJDK_TARGET_CPU))
+  ifeq (cross,$(COMPILE_TYPE))
+    export SYSROOT_CFLAGS := @SYSROOT_CFLAGS@
```

Comment:
This pull request doesn't appear to define `SYSROOT_CFLAGS`: where will the definition come from?

Reply:
`SYSROOT_CFLAGS` is defined by the JDK11 extension itself in which case we need to specify the path of mounted OS image with `--with-sysroot`.  We export `SYSROOT_CFLAGS` to enable the cross-compiler to locate the correct library/header path on the target system.
e.g.
```
bash configure
--disable-warnings-as-errors
--enable-ddr=no
--with-build-jdk=/.../bootjdk11_openj9
--openjdk-target=riscv64-unknown-linux-gnu
--with-sysroot=/.../fedora_riscv_mnt <--- the path to the mounted OS image
...
--with-freemarker-jar=/.../freemarker.jar
```
and the cross-compiler will pick it up later like:
```
riscv64-unknown-linux-gnu-g++ --sysroot=/.../fedora_riscv_mnt  -I. -I../../include
...-funroll-loops  -o ConcurrentGCSATB.o ConcurrentGCSATB.cpp
``

Reply:
`SYSROOT_CFLAGS` should already be defined in `spec.gmk`: it should only need to be exported here (not redefined).

Reply:
As suggested above, I will verify `export SYSROOT_CFLAGS` by compiling a cross-build to see whether it works as expected.

Reply:
I've compiled a cross-build with `export SYSROOT_CFLAGS` which works good on RISC-V. So updated to `export SYSROOT_CFLAGS`.